File: app/src/main/java/test/platzi/com/raian/com/org/simpledynamit/model/country/ResultOpenAQCountry.java

package test.platzi.com.raian.com.org.simpledynamit.model.country;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ResultOpenAQCountry {

    @SerializedName("results")
    @Expose
    private List<Result> results = null;

    public List<Result> getResults() {
        return results;
    }

    public void setResults(List<Result> results) {
        this.results = results;
    }

}

File: app/src/main/java/test/platzi/com/raian/com/org/simpledynamit/model/country/ResultOpenAQCountry.kt
package test.platzi.com.raian.com.org.simpledynamit.model.country

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ResultOpenAQCountry {

    @SerializedName("results")
    @Expose
    var results: List<Result>? = null

}
